ember-toucan-core/src/components/form/file-input-field.hbs

```diff
@@ -0,0 +1,110 @@
+<div
+  class="flex flex-col {{if @isDisabled 'text-disabled'}}"
+  data-root-field={{if @rootTestSelector @rootTestSelector}}
+>
+  <Form::Field as |field|>
+    <field.Label for={{field.id}} data-label>
+      {{@label}}
+      {{#if @hint}}
+        <field.Hint data-hint>{{@hint}}</field.Hint>
+      {{/if}}
+
+      <field.Control class="flex">
+        <div
+          class="bg-overlay-1 shadow-focusable-outline focus-within:shadow-focus-outline relative inline-flex h-8 w-full flex-wrap items-center rounded-sm"
+        >
+          <input
+            aria-describedby="{{if @error field.errorId}}"
+            {{! template lint thinks this input is a button, and so doesn't not support aria-invalid }}
+            {{! template-lint-disable no-unsupported-role-attributes }}
+            aria-invalid={{if @error "true"}}
+            class="text-titles-and-attributes focus:outline-none absolute m-1 h-6 w-0 cursor-pointer bg-transparent p-1
+              {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}
+              {{if @error 'shadow-error-outline' 'shadow-focusable-outline'}}"
+            disabled={{@isDisabled}}
+            id={{field.id}}
+            type="file"
+            accept={{this.accept}}
+            multiple={{this.multiple}}
+            {{! template-lint-disable no-bare-strings no-inline-styles  }}
+            style="clip: rect(1px, 1px, 1px, 1px);"
+            ...attributes
+            {{on "change" this.onChange}}
+          />
+          <div
+            class="text-titles-and-attributes inline-flex w-full cursor-pointer items-center gap-x-2 pl-2"
```

Comment:
I'm curious about the `pl-2` here.  Is that to offset the icon from the left-side of the box?  If so, should we remove this class and instead apply padding to the wrapping div (line 12/13) above instead?